IdentityCore/src/configuration/MSIDNetworkConfiguration.h

```diff
@@ -24,11 +24,9 @@
 #import <Foundation/Foundation.h>
 
 
-@interface MSIDNetworkConfiguration : NSObject <NSCopying>
+@interface MSIDNetworkConfiguration : NSObject
```

Comment:
Should MSIDNetworkConfiguration be singleton? If no, how are we going to access it in the code?

Reply:
Both it's properties are class properties with setters and getters.
So, you'd just use MSIDNetworkConfiguration.timeout and MSIDNetworkConfiguration.retryCount

Reply:
https://developer.apple.com/videos/play/wwdc2016/405/  at 5:07 for reference.